.github/workflows/go-test.yml

```diff
@@ -16,6 +16,6 @@ concurrency:
 
 jobs:
   go-test:
-    uses: ipdxco/unified-github-workflows/.github/workflows/go-test.yml@v1.0
+    uses: filecoin-project/unified-github-workflows/.github/workflows/go-test.yml@v1.0
```

Comment:
I am curious as to why only `go-test` workflow is updated. Do we want to consistently use the workflows  from `filecoin-project` fork?

Reply:
Not necessarily, we only need to use workflows from the fork in `filecoin-project` in the workflows where we want to use custom GitHub runners.

The fork and the origin are kept identical automatically. The fork only exists to overcome the limitation that disallows usage of self-hosted runners in reusable workflows that live outside the org.